include/mxnet/c_api.h

```diff
@@ -55,7 +55,8 @@ extern "C" {
 #endif
 
 /*! \brief manually define unsigned int */
-typedef unsigned int mx_uint;
+typedef int64_t mx_int64;
```

Comment:
Just a nit but,
As per the comment
`/*! \brief manually define unsigned int */`
Line
`typedef int64_t mx_int64;` should be above it I guess, as it is not `unsigned`.